libs/blocks/merch-card/merch-card.js

```diff
@@ -119,25 +121,49 @@ const isParagraphTag = (tagName) => tagName === 'P';
 
 const parseContent = (el, merchCard) => {
   const innerElements = [
-    ...el.querySelectorAll('h2, h3, h4, h5, p, ul'),
+    ...el.querySelectorAll('h2, h3, h4, h5, p, ul, em'),
   ];
-  const bodySlot = createTag('div', { slot: 'body-xs' });
+  let bodySlotName = 'body-xs';
+  let headingMCount = 0;
+  if (merchCard.variant === MINI_COMPARE_CHART) {
+    bodySlotName = 'body-m';
+  }
+  const bodySlot = createTag('div', { slot: bodySlotName });
 
   innerElements.forEach((element) => {
     const { tagName } = element;
+    if (element.innerHTML.trim() === '') {
+      element.remove();
+      return;
+    }
+    if (tagName === 'EM' && !element.querySelector('a')) {
```

Comment:
Can you target this in the test somehow for full coverage? Maybe add an empty inner row to one of the test-*.html? 🤷 
or maybe ... `/* c8 ignore next 3 */` above

Reply:
![Screenshot 2024-01-24 at 12 18 59 PM](https://github.com/adobecom/milo/assets/1179549/b09af4a7-c6a6-4b27-9f87-a43e290ba4fb)
No worries, it's taken care of.